R/unreachable_code_linter.R

```diff
@@ -43,6 +72,30 @@ unreachable_code_linter <- function() {
     ]
     /following-sibling::*[1]
   "
+  xpath_if_while <- "
+  //WHILE[following-sibling::expr[1]/NUM_CONST[text() = 'FALSE']]/following-sibling::expr[2]
+  |
+  //IF[following-sibling::expr[1]/NUM_CONST[text() = 'FALSE']]/following-sibling::expr[2]
+  "
+
+  xpath_else <- "
+  //IF[following-sibling::expr[1]/NUM_CONST[text() = 'TRUE']]
+  /following-sibling::ELSE/following-sibling::expr[1]
+  "
+
+  handle_inline_conditions <- function(lints) {
+    lints <- lapply(
+      lints,
+      FUN = function(x) {
+        if (xml2::xml_name(xml2::xml_child(x)) == "OP-LEFT-BRACE") {
```

Comment:
`xml_name` and `xml_find_first` are in our namespace, no need to qualify:

https://github.com/r-lib/lintr/blob/c8f00746501e041d2516da58ccca3e054cf305f1/NAMESPACE#L170-L173